app/src/main/java/com/peacecorps/pcsa/support_services/SupportServices.java

```diff
@@ -0,0 +1,31 @@
+package com.peacecorps.pcsa.support_services;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.peacecorps.pcsa.R;
+
+public class SupportServices extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_support_services);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
+        fab.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
```

Comment:
It is like this. The code is open and shared. So the app should always be in a consistent and clean state. This is just adding some blank activities and hoping to fill them with their correct functionality in the future. If you are developing the app individually, this approach may be good. But since this is open source, it is better to create them as you go. So the code repo will always have a good functioning, clean code. Furthermore, things like Floating Action Button has no use in future too (As I saw in the mockup). So there is no use in keeping the unnecessary code to remove in the future. You have just added the auto generated blank activities. You can always do that in a jiffy. So adding these activities early won't be a significant advantage. If you want to keep the architecture documented, do not use the code. Create an architecture diagram or class diagram. So we can add them in the docs folder or wiki when reviewed.